assets/js/mixins/formatter.js

```diff
@@ -289,6 +289,55 @@ export default {
       }
       return price;
     },
+    fmtRepeatingPlansUTC: function (plans, toUTC) {
+      return plans.map((plan) => {
+        let [time, dayOffset] = this.fmtDayHourMinute(plan.time, toUTC);
+
+        let newWeekdays = plan.weekdays.map(function (weekday) {
+          return (weekday + dayOffset + 7) % 7;
+        });
+
+        return {
+          ...plan,
+          time: time,
+          weekdays: newWeekdays,
+        };
+      });
+    },
+    fmtDayHourMinute: function (timeString, toUTC) {
```

Comment:
Mir ist nicht klar wofür wir das brauchen. Aktuell wird das nur einmal aufgerufen (`toUTC=true`), bei der Rückgabe wird nur der erste Wert gelesen.

Generellere Frage: Wieso müssen wir das überhaupt machen? Wir können hier für den Start sicher die Annahme Treffen, dass Server und Client sich in der gleichen Zeitzone befinden. Heißt, wir müssen auf der Server-Seite immer in lokaler Zeitzone parsen. Spätestens mit der Datumsgrenze wird das ja sonst ohnehin etwas problematisch.

Wenn wir hier Probleme bekommen (Nutzer kann die Zeitzoneneinstellung auf dem OS auf dem evcc läuft nicht ändern, ...) könnten wir immer noch eine Zeitzonen-Einstellung einführen, die wir fürs Parsing in Go nutzen.